recipes/cython_gsl/meta.yaml

```diff
@@ -1,19 +1,21 @@
-{% set name = "CythonGSL" %}
+{% set name = "cython_gsl" %}
```

Comment:
The package is called cython_gsl in the project's setup.py so I used that. Does it need to match?